src/peerGroup.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 var debug = require('debug')('bitcoin-net:peergroup')
-var dns = require('dns')
+try { var dns = require('dns') } catch (err) {}
```

Comment:
Sorry, I use mean.io and it's using webpack (earlier version was browserify).
I could fix the (net and) dns issue by adding 
node: {
        'net': 'empty',
        'dns': 'empty'
    }
to the webpack.config.js